File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Bass/BAS0001U00FbxDodobuHyeunDataValidatingInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Bass
{
    [Serializable]
	public class BAS0001U00FbxDodobuHyeunDataValidatingInfo
	{
		private String _codeName;
		private String _sortKey;

		public String CodeName
		{
			get { return this._codeName; }
			set { this._codeName = value; }
		}

		public String SortKey
		{
			get { return this._sortKey; }
			set { this._sortKey = value; }
		}

		public BAS0001U00FbxDodobuHyeunDataValidatingInfo() { }

		public BAS0001U00FbxDodobuHyeunDataValidatingInfo(String codeName, String sortKey)
		{
			this._codeName = codeName;
			this._sortKey = sortKey;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/bass/BAS0001U00FbxDodobuHyeunDataValidatingInfo.java
package nta.med.data.model.ihis.bass;

public class BAS0001U00FbxDodobuHyeunDataValidatingInfo {
	private String codeName;
	private Double sortKey;
	public BAS0001U00FbxDodobuHyeunDataValidatingInfo(String codeName,
			Double sortKey) {
		super();
		this.codeName = codeName;
		this.sortKey = sortKey;
	}
	public String getCodeName() {
		return codeName;
	}
	public void setCodeName(String codeName) {
		this.codeName = codeName;
	}
	public Double getSortKey() {
		return sortKey;
	}
	public void setSortKey(Double sortKey) {
		this.sortKey = sortKey;
	}
	

}
